includes/agentic-commerce/class-wc-stripe-agentic-commerce-product-list-table.php

```
@@ -0,0 +1,425 @@
+	 * @since 10.9.0
+	 * @param string $column  Column key being rendered.
+	 * @param int    $post_id Row's post ID.
+	 * @return void
+	 */
+	public function render_column( $column, $post_id ): void {
+		if ( self::COLUMN_KEY !== $column ) {
+			return;
+		}
+
+		$product = wc_get_product( (int) $post_id );
+		if ( ! $product instanceof WC_Product || ! in_array( $product->get_type(), WC_Stripe_Agentic_Commerce_Product_Exclusion::get_supported_types(), true ) ) {
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'WC_Stripe_Agentic_Commerce_Product_Exclusion' in method 'render_column'. (undefined)

(StaticAccess)